uat/testing-features/src/main/java/com/aws/greengrass/steps/NetworkUtilsSteps.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.steps;
+
+import com.aws.greengrass.platforms.Platform;
+import com.aws.greengrass.testing.model.ScenarioContext;
+import io.cucumber.guice.ScenarioScoped;
+import io.cucumber.java.After;
+import io.cucumber.java.ParameterType;
+import io.cucumber.java.en.When;
+import lombok.extern.log4j.Log4j2;
+
+import java.io.IOException;
+import java.util.concurrent.atomic.AtomicBoolean;
+import javax.inject.Inject;
+
+@Log4j2
+@ScenarioScoped
+public class NetworkUtilsSteps {
+
+    private final ScenarioContext scenarioContext;
+
+    private final AtomicBoolean mqttConnectivity = new AtomicBoolean(true);
+
+    @Inject
+    public NetworkUtilsSteps(ScenarioContext scenarioContext) {
+        this.scenarioContext = scenarioContext;
+    }
+
+    /**
+     * Convert offline/online string to boolean.
+     *
+     * @param value the string of offline/online value
+     * @throws RuntimeException on invalid value
+     */
+    @SuppressWarnings("PMD.UnnecessaryAnnotationValueElement")
+    @ParameterType(value = "offline|Offline|OFFLINE|online|Online|ONLINE")
+    public boolean connectivityValue(String value) {
+        switch (value) {
+            case "offline":
+            case "Offline":
+            case "OFFLINE":
+                return false;
+
+            case "online":
+            case "Online":
+            case "ONLINE":
+                return true;
+
+            default:
+                throw new RuntimeException("Invalid connectivity value " + value);
```

Comment:
I think **log.error** is necessary.

Reply:
added in f820a556c989ecf5ecb2a117bc62f3124ae7d774